hrui/js/src/hrui/main.js

```diff
@@ -65,17 +65,15 @@
     /**
      * Amends the contact page and the contact form
      */
-    function amendContactPageAndForm (e) {
-      if (CRM.formName === 'contactForm' || CRM.pageName === 'viewSummary') {
```

Comment:
isn't this check important? Why do we want to (try to) do this DOM manipulation in any page now?

Reply:
Yes this should not be removed, reverted.